insights/parsers/mount.py

```diff
@@ -275,3 +278,53 @@ def search(self, **kwargs):
             (list): The list of mount points matching the given criteria.
         """
         return keyword_search(self.rows, **kwargs)
+
+
+@parser(Specs.mounts)
+class ProcMount(Mount):
```

Comment:
Better rename it to `ProcMounts` to align with the actual file.

Reply:
@xiangce - Acknowledged. Changes done in the latest commit.